x/blob/module_simulation.go

```diff
@@ -0,0 +1,67 @@
+package blob
+
+import (
+	"math/rand"
+
+	"github.com/celestiaorg/celestia-app/testutil/sample"
+	bloblsimulation "github.com/celestiaorg/celestia-app/x/blob/simulation"
+	"github.com/celestiaorg/celestia-app/x/blob/types"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	simappparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/authz/codec"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+)
+
+// avoid unused import issue
+var (
+	_ = sample.AccAddress
+	_ = bloblsimulation.FindAccount
+	_ = simappparams.StakePerAccount
+	_ = simulation.MsgEntryKind
+	_ = baseapp.Paramspace
```

Comment:
[Question]
Question please, if these are not used, why do we need to import them?